src/agentscope/memory/memory.py

```diff
@@ -63,7 +68,7 @@ def delete(self, index: Union[Iterable, int]) -> None:
     @abstractmethod
     def load(
         self,
-        memories: Union[str, dict, list],
```

Comment:
Suggest to be more specific, e,g. `Union[str, List[MessageBase], MessageBase]`

Similar problem with the typing of the return values in export interface.